src/core/Log_semgrep.ml

```diff
@@ -0,0 +1,56 @@
+let src = Logs.Src.create "semgrep"
+
+module Log = (val Logs.src_log src : Logs.LOG)
+
+let help_msg =
+  {|
+If you want to show the logs of a particular library
+(e.g., the semgrep targeting library), you'll need to adjust the
+SEMGREP_LOG_SRCS environment variable as in
+
+  export SEMGREP_LOG_SRCS="semgrep.targeting" semgrep ... --debug
+
+or for osemgrep you can use it with any log level as in
+
+   export SEMGREP_LOG_SRCS="semgrep.targeting" ./bin/osemgrep ... --verbose
+
+You can see the list of possible libraries above in this log as in
+...
+[00.04][DEBUG](default): Skipping logs for semgrep.targeting
+...
+[00.04][DEBUG](default): Skipping logs for commons.pcre
+...
+For more information, See
+https://www.notion.so/semgrep/Logging-in-semgrep-semgrep-core-osemgrep-67c9046fa53744728d9d725a5a244f64
```

Comment:
Users will see this too.

Reply:
If they use --debug yes, but I don't think it's a problem. --debug is not designed for users.